grails-app/services/org/pih/warehouse/core/DocumentService.groovy

```diff
@@ -934,6 +931,43 @@ class DocumentService {
         }
     }
 
+    void generateInventoryTemplate(OutputStream outputStream, List<Map> data) {
```

Comment:
I would prefer if we did something more generic here and expanded the generateExcel() to handle formats and other features.

Reply:
@jmiranda that's the part I was not sure before (one the first sight). It looked very similar to the code that already is there. So I agree that this could be more generic.